dapp/src/components/AccountListener.js

```diff
@@ -410,15 +438,33 @@ const AccountListener = (props) => {
 
   useEffect(() => {
     let balancesInterval
-    if (contracts && userActive === 'active' && isCorrectNetwork(chainId)) {
+    if (
+      account &&
+      contracts &&
+      contracts.ogn.provider &&
```

Comment:
@nickgeoca just curious was there a case where the contracts have initialised and the `contracts.ogn.provider` was undefined or null?

Reply:
I saw it in the [loadData](https://github.com/OriginProtocol/origin-dollar/blob/nkg/reduceAlchemyCalls2/dapp/src/components/AccountListener.js#L153-L157) fn way back and kept it in for consistency.

Reply:
They're both using the provider to grab data

Reply:
ok coo makes sense